components/campaigns-open-view/MessageItem.tsx

```
@@ -11,94 +11,119 @@ interface MessageItemProps {
-              }}
-              source={{
-                uri: v.video_data.url,
-              }}
-              resizeMode={ResizeMode.CONTAIN}
-              useNativeControls
-            />
-          )}
-        </View>
+      {msg.attachments?.data?.map((v) => {
+        return (
+          <View>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)